portfolio/src/main/webapp/calendar.js

```diff
@@ -27,7 +27,9 @@ class CalendarEvent {
  * CalendarEvent from this information. Adds the new event to the
  * event list.
  */
-function createNewCalendarEvent() {
+function createNewCalendarEventUserInput() {
+  setWarningsToHidden();
```

Comment:
It wasn't clear to me why we have to call this method here. Is it to clear existing warnings? Consider adding some comment.

Reply:
Updated.

Reply:
Updated. Also, after I changed the visibility toggling to jQuery, I don't need this setWarningsToHidden function anymore.